janus-idp/backstage-showcase
# fix(admin): added RBAC tab display condition

## Description

Guest users get empty RBAC content while permission not exists.

## Which issue(s) does this PR fix

- Fixes [RHIDP-1333](https://issues.redhat.com/browse/RHIDP-1333): RBAC tab is shown even for the guest users who don't have permission 

## PR acceptance criteria

Please make sure that the following steps are complete:


*Unit test and e2e test will be handled in a separate PR.

## How to test changes / Special notes to the reviewer
Login with both guest and GitHub users and observe RBAC tab for GitHub users.

Screen video


https://github.com/janus-idp/backstage-showcase/assets/26255262/37f71b05-7f69-4d30-92d3-cfe1fc5ef627
